servers/shm_test/main.c

```diff
@@ -0,0 +1,52 @@
+
+#include <resea/ipc.h>
+#include <resea/printf.h>
+#include <string.h>
+
+#define MSG "hello task b"
+
+int main() {
```

Comment:
Nit: A return value from `main` have no not meaning on Resea. This line should be `void main(void)`.